docs/integrations/webhooks/inngest.mdx

```diff
@@ -0,0 +1,215 @@
+
+<Images
+  width={1906}
+  height={1376}
+  src="/docs/images/integrations/inngest/webhook-transformation-template.webp"
+  alt="The Webhooks page in the Clerk Dashboard showing the Inngest transformation template. Red arrows point to the Transformation Template tab, the Inngest template, and the Connect to Inngest button."
+/>
+
+A popup window will appear to complete the setup. Select **Approve** to create the webhook.
+
+<Images
+  width={1880}
+  height={1448}
+  src="/docs/images/integrations/inngest/webhook-endpoint-connected.webp"
+  alt="The Webhooks page in the Clerk Dashboard showing a connected Inngest account. A red arrow points to the Connected button."
+/>
+
+After the popup window disappears, the Webhooks page will now display **Connected** with the webhook URL underneath. There is one more step to complete setup.
+
+<Images
+  width={1194}
+  height={1520}
+  src="/docs/images/integrations/inngest/inngest-permissions-dialog.webp"
+  alt="The Inngest permissions popup window showing the Approve button."
+/>
+
+To complete the setup, scroll down and select **Create**.
+
+<Images
+  width={1906}
+  height={1376}
+  src="/docs/images/integrations/inngest/webhook-create.webp"
+  alt="The Webhooks page in the Clerk Dashboard showing the end of the page to create a new endpoint. A red arrow points to the Create button."
+/>
+
+
+You'll be redirected to the new endpoint. In your Inngest dashboard, you will see a new webhook created in your account's [production environment](https://app.inngest.com/env/production/manage/webhooks).
+
+
+## Viewing webhook events within Inngest
+
+After setup, as webhook events are sent from Clerk to Inngest, new `clerk `events will appear  in your Inngest dashboard. Event names will be the prefixed `clerk/` followed by the event name. See [webhook events](/docs/integrations/webhooks#supported-webhook-events) for a full list.
+
+<Images
+  width={1052}
+  height={860}
+  src="/docs/images/integrations/inngest/inngest-clerk-events.webp"
+  alt="The Events page in the Inngest Dashboard showing a list of Clerk events."
+/>
+
+## Creating a function to sync a new user to a database
+
+With Inngest already set up in your codebase, you can now use these events as triggers for your functions.
+
+The code below features the `inngest.createFunction` method, which inserts a new user into the database. It will be triggered whenever `clerk/user.created` event occurs.
+
+
+```typescript
+const syncUser = inngest.createFunction(
+ { id: 'sync-user-from-clerk' },
+ { event: 'clerk/user.created' }, // ← This is the function's triggering event
+ async ({ event }) => {
+   const user = event.data; // The event payload's data will be the Clerk User json object
+   const { id, first_name, last_name } = user;
+   const email = user.email_addresses.find(e =>
+     e.id === user.primary_email_address_id
+   ).email
+   await database.users.insert({ id, email, first_name, last_name })
+ }
+)
+```
```

Comment:
I think we can make this a little clearer that this in the customer's codebase. You can use something like `filename="src/inngest/sync-user.ts"` on line 88 after the language as well. 

Also, having not used Inngest and just reading this I am wondering if the 'id' value is just a label as it appears to be or if it should be an id string of some sort.

Reply:
Good idea and feedback. I've changed the description paragraph, added the file name, and a comment about the `id` ✅ Thank you!